### ISSUE TITLE
SQL Injection? 

### ISSUE THREAD

**Issue by Ed-zo** [danielhros/lora-application-server#15]
Neviem ci sa jedna o dolezity projekt ale mnoho SQL prikazov v routes zlozke dovoluje vykonat SQL Injection. Tu je jeden z nich. Ak to nie je dolezity projekt, issue mozes closnut.

https://github.com/danielhros/lora-application-server/blob/94fb7add9321790c2e47c9d2c40754e189afa437/routes/device.js#L168-L174

**Comment by danielhros** [comment]
Áno, viem o tomto probléme. Pri projekte to však nebolo to najdôležitejšie na čo som sa potreboval sústrediť a je v pláne túto issue vyriešiť. Diky za upozornenie, dopyty sú nateraz chránené autentifikáciou.
